docs/processes/release/tasks/legal/find_textfiles

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/perl -w 
+#
+# Copyright (C) 2012, 2017: Unicode, Inc. and others.
+# License & terms of use: http://www.unicode.org/copyright.html
```

Comment:
@yumaoka I suppose I should just delete this file?
- Author is listed below as Umesh Nair.
- Copyright notice years I based on https://sites.google.com/site/icusite/system/app/pages/admin/revisions?wuid=wuid:gx:305064f7a236995f which has two versions of the script, uploaded by you.